phoenix-core/src/main/java/org/apache/phoenix/compile/FromCompiler.java

```diff
@@ -1121,46 +1124,55 @@ public PSchema resolveSchema(String schemaName) throws SQLException {
             return null;
         }
     }
-    
+
     private static class ProjectedTableColumnResolver extends MultiTableColumnResolver {
         private final boolean isLocalIndex;
+        // We must handle the local index data tables separately
+        protected final ListMultimap<String, TableRef> localIndexDataTableMap = ArrayListMultimap.<String, TableRef> create();
```

Comment:
nit: `ArrayListMultimap.create()` would do?

Reply:
Yes, but we use the same initialization for tableMap, and I wanted to keep as similar as possible.
We could change it in both places, but that would add even more noise to the patch.